Documentation/disaster-recovery.md

```diff
@@ -106,3 +106,102 @@ kubectl create -f operator.yaml
 ```
 
 The operator will automatically add more mons to increase the quorum size again, depending on the `monCount`.
+
+# Bring orphaned rook/ceph cluster back to Kubernetes
+
+## What this section trying to resolve
+
+1. You had a running rook/ceph cluster, suddenly your Kubernetes environment exploded, you have to start a new Kubernetes environment and put your existing rook/ceph cluster back.
+2. You are migrating your existing rook/ceph cluster to a new Kubernetes environment, downtime can be tolerated.
+
+In author's situation,  `etcd` data of running Kubernetes cluster is nuked and have no backup, all OSDs are using bluestore backend.
+
+### Prerequisites
+
+1. A working Kubernetes cluster without rook
+2. Previous rook/ceph cluster is intact, which means you have at least one ceph mon data intact, and all your OSD data is intact.
+
+Tested in `rook/ceph:v1.0.3` and `ceph/ceph:v14.2.1-20190430`.
+
+### TL;DR for Steps below
+1. Start a new and clean rook cluster, with old `CephCluster` & friends. Shut it down when it seems working(as a brand new cluster).
```

Comment:
Elaborate on "& friends". Should the CRDs match the previous clusters' exactly? Should any resources be omitted?

Reply:
Need space before left paren

Reply:
rook -> Rook

Reply:
This seems like 2 steps. 
1. Start new cluster.
1. Shut the new cluster down when it has been created successfully.

Reply:
fixed